config/config.go

```diff
@@ -138,6 +153,9 @@ type Config struct {
 	RemoteWriteConfigs []*RemoteWriteConfig `yaml:"remote_write,omitempty"`
 	RemoteReadConfigs  []*RemoteReadConfig  `yaml:"remote_read,omitempty"`
 
+	// experimental tracing configuration section
```

Comment:
We shouldn't be adding anything to the config file. Adding Jaeger is temporary, and not a part of the semantics of the Prometheus.

Reply:
So you want it completely controlled by env variables?

Reply:
I never said that, how does it work out of the box?

Reply:
It can be done completely with env variables out of the box. If that's the better approach I'm happy to take the config bit out

Reply:
That sounds okay, as long as we're never touching them ourselves.

Reply:
Now configuration is sourced entirely from environment variables and everything is off by default.